update.js

```diff
@@ -12,6 +12,12 @@ packageJsonOpenLens.build.artifactName = "${productName}-${buildVersion}-${arch}
 packageJsonOpenLens.build.linux.artifactName = packageJsonOpenLens.build.linux.artifactName.replace("${version}", "${buildVersion}");
 packageJsonOpenLens.build.win.artifactName = "OpenLens.Setup.${buildVersion}.${ext}";
 packageJsonOpenLens.build.npmRebuild = true;
+packageJsonOpenLens.copyright = [
+    packageJsonOpenLens.copyright,
+    '',
+    'Binary application builds by MuhammedKalkan/OpenLens',
+    'Muhammed Kalkan, Jan-Otto Kröpke, Ebby Peter, Xaver Lohmüller'
```

Comment:
```suggestion
    'Binary application builds @ MuhammedKalkan/OpenLens',
    'by Muhammed Kalkan, Jan-Otto Kröpke, Ebby Peter, Xaver Lohmüller'
```

@MuhammedKalkan like this?

Reply:
Yes